src/Guild.js

```diff
@@ -25,6 +25,35 @@ class Guild {
     autoBind(this);
   }
 
+  guildRoleCreate (roleId) {
+    this.roles[this.roles.name] = roleId;
+  }
+
+  guildGetKeyByValue (value) {
+    return Object.keys(this.roles).find(key => this.roles[key] === value);
+  }
+
+  guildUpdateRoleName (roleId) {
+    const oldKey = this.guildGetKeyByValue(roles.id);
```

Comment:
`roles.id` is undefined. I assume you meant to use `roleId` instead.

Reply:
Yes. Fixed